app/services/activities/import_from_csv.rb

```diff
@@ -67,7 +69,7 @@ def update_activity(row, index)
       elsif row["Parent RODA ID"].present?
         add_error(index, :parent_id, row["Parent RODA ID"], I18n.t("importer.errors.activity.cannot_update.parent_present")) && return
       else
-        updater = ActivityUpdater.new(row, @organisation)
+        updater = ActivityUpdater.new(row: row, user: @uploader, delivery_partner_organisation: @delivery_partner_organisation)
```

Comment:
For consistency, it's probably worth reordering the parameters here to match the `ActivityCreator.new` call above.